cloudbuild.yaml

```diff
@@ -33,7 +33,7 @@ steps:
       - 'JAVA_TOOL_OPTIONS="-Xmx3g"'
       - 'GRADLE_OPTS="-Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=8 -Dkotlin.incremental=false"'
       - 'BRANCH_NAME=$BRANCH_NAME'
-      - 'EXAMPLE_AS_ROOT=false'
+      - 'EXAMPLE_EMBEDDING_METHOD="maven"'
```

Comment:
We actually don't use cloud build anymore, so this whole file can be removed instead of updating it.

Reply:
also it would not work anyway I think, it does not appear the env variable is read anywhere.

Reply:
It is picked up, I suppose we could use this on Bitrise instead of generating a local.properties too then.

Reply:
Yes, if local properties or environmental variables a defined, it will use a composite build.
 
I think we should use bitrise.yaml configuration instead of generating local.properties. Where we are doing it (generating I mean), by the way? Can not find this code in the project (also don't have the rights to edit bitrise)

Reply:
the problem AFAIK is there are multiple Bitrise projects for one project which will all try to grab the same yaml file even though they need to do different things. If you have a solution for this, feel free, else it's the "generate text file" step in Bitrise (as `dennis.splendo` you (now) have access there)